README.md

```diff
@@ -64,7 +89,13 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+      function doSomething(thingToDo){
+
+          thingToDo();
+      }
+      doSomething(function(){
+        console.log("like this?")
+      })
```

Comment:
Yes, but I'd recommend putting the console logging function on a different line:

```
function whatever(){
  console.log("like this?")
}
doSomething(whatever);
```